app/controllers/shared/api.php

```diff
@@ -114,6 +114,32 @@
 
     $deletes->setProject($project);
     $database->setProject($project);
+
+    $useCache = $route->getLabel('cache', false);
+    if ($useCache) {
+            $key = md5($request->getURI() . $request->getServer('query_string'));
+            $cache = new Cache(new Filesystem(APP_STORAGE_CACHE . DIRECTORY_SEPARATOR . 'app-' . $project->getId()));
+            $data = $cache->load($key, 60 * 60 * 24 * 30 * 1);
+        if (!empty($data)) {
+            $cacheLog = $dbForProject->getDocument('cache', $key);
```

Comment:
Don't we need to skip Authorization here?

Reply:
fixed